spec/fortnox/api/mappers/base/from_json_spec.rb

```diff
@@ -24,10 +24,10 @@
       {
         'Product' => {
           '@url': 'someurl@example.com',
-          'Name': 'Ford Mustang',
-          'VAT': 30_000,
-          'Categories': [{ 'Name' => 'Cars', 'ID' => '1' }, { 'Name' => 'Fast cars', 'ID' => '2' }],
-          'Designer': { 'Name' => 'John Najjar', 'ID' => '23' }
+          Name: 'Ford Mustang',
```

Comment:
Ugh ... I know what it's doing, but I do like it when the same convention is used in every case - at least locally 😄 (It's fine ... I just wanted to complain a bit)